kinder/hack/update-workflows.sh

```diff
@@ -33,7 +33,7 @@ PATH_WORKFLOWS="${PATH_WORKFLOWS:-./ci/workflows/}"
 
 # set test-infra path
 TEST_INFRA_SIG_DIR="config/jobs/kubernetes/sig-cluster-lifecycle"
-PATH_TEST_INFRA="${PATH_TEST_INFRA:-"$HOME/go/src/k8s.io/test-infra/$TEST_INFRA_SIG_DIR"}"
+PATH_TEST_INFRA="${PATH_TEST_INFRA:-"${GOPATH}/src/k8s.io/test-infra/${TEST_INFRA_SIG_DIR}"}"
```

Comment:
would GOPATH exist in newer versions of Go and new systems?
i think they were planning to remove it?